resource/decker/support/js/mathjax.js

```diff
@@ -100,19 +100,19 @@ export function configureMathJax() {
     options: {
       // skipHtmlTags: { "[+]": ["details"] },
       enableMenu: a11y,
-      enableExplorer: a11y,
+      //      enableExplorer: a11y,
       menuOptions: {
         settings: {
           enrich: a11y, // true to enable semantic-enrichment
-          collapsible: false, // true to enable collapsible math
-          speech: a11y, // true to enable speech generation
-          braille: a11y, // true to enable Braille generation
-          assistiveMml: false, // true if hidden assistive MathML should be generated for screen readers
+          //          collapsible: false, // true to enable collapsible math
+          //          speech: a11y, // true to enable speech generation
+          //          braille: a11y, // true to enable Braille generation
+          //          assistiveMml: false, // true if hidden assistive MathML should be generated for screen readers
         },
```

Comment:
disable anything not really necessary. benchmark timings with MafI decks and pages